controllers/subnamespace_controller.go

```diff
@@ -29,6 +29,7 @@ type SubNamespaceReconciler struct {
 //+kubebuilder:rbac:groups=accurate.cybozu.com,resources=subnamespaces/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=accurate.cybozu.com,resources=subnamespaces/finalizers,verbs=update
 //+kubebuilder:rbac:groups="",resources=namespaces,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups="",resources=resourcequotas,verbs=get;list;watch;create;update;patch;delete
```

Comment:
The subnamespace controller does not require any privileges for ResourceQuotas.
That should be configured through a Helm variable.